### DISCUSSION TITLE
import webpack library in another app the request worker return 404

### DISCUSSION THREAD

**Discussion by ckken** [webpack/webpack#14904] [General]
# Bug report
![image](https://user-images.githubusercontent.com/3890513/144699080-0741cc00-525f-4707-a8ca-3a231005c605.png)

```
// webpack library export module umd & esm
import logger from './logger'
import mod from './mod'
console.log('import.meta.url', import.meta.url, new URL('src/worker.ts', import.meta.url))
const worker = new Worker(new URL('src/worker.ts', import.meta.url))
// const worker = new Worker(new URL('src/worker.ts', import.meta.url), {
//   type: 'module',
// })

const App = () => {
  mod('this is mod')
  worker.postMessage({
    question: 'The Answer to the Ultimate Question of Life, The Universe, and Everything.',
  })
  worker.onmessage = ({data: {answer}}) => {
    logger.info('[main thead]', answer)
  }
}
export default App

```

**What is the current behavior?**
return 404

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
return web worker console

**Other relevant information:**
webpack version:  5.64.4
Node.js version: 16.13.0
Operating System: window
Additional tools:

**Comment by ckken** [comment]
![image](https://user-images.githubusercontent.com/3890513/144699096-557a6561-3609-4ff1-9732-dbf8710b3e02.png)
library output files

**Comment by alexander-akait** [comment]
Can you provide reproducible test repo?

**Reply by ckken** [reply]
https://github.com/ckken/webpack-library-web-worker  thanks for review @alexander-akait

**Reply by alexander-akait** [reply]
Duplicate https://github.com/webpack/webpack/issues/12719
